com.chartboost.mediation/Runtime/Utilities/Constants.cs

```diff
@@ -0,0 +1,12 @@
+namespace Chartboost.Utilities
+{
+    public static class Constants
+    {
+        public static class BannerSize
+        {
+            public static readonly (float, float) STANDARD = (320, 50);
+            public static readonly (float, float) MEDIUM = (300, 250);
+            public static readonly (float, float) LEADERBOARD = (728, 90);
+        }
+    }
+}
```

Comment:
Nit: There are a few EOL, may want to adjust the IDE to auto add them on save. 👍🏼